examples/flows/standard/conditional-flow-for-if-else/flow.dag.yaml

```diff
@@ -0,0 +1,47 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
+inputs:
+  question:
+    type: string
+    default: What is Prompt flow?
+outputs:
+  answer:
+    type: string
+    reference: ${generate_result.output}
+nodes:
+- name: content_safety_check
+  type: python
+  source:
+    type: code
+    path: content_safety_check.py
+  inputs:
+    text: ${inputs.question}
+- name: llm_result
+  type: python
+  source:
+    type: code
+    path: llm_result.py
+  inputs:
+    question: ${inputs.question}
+  activate:
```

Comment:
do we have example of non-boolean case, such as boolean/ int, which is supported in VSCode

Reply:
Next PR will add another example flow for switch scenario that activate config contains string/int.